vsintegration/src/FSharp.Editor/CodeFix/AddNewKeywordToDisposableConstructorInvocation.fs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace rec Microsoft.VisualStudio.FSharp.Editor
+
+open System.Composition
+open System.Collections.Immutable
+open System.Threading
+open System.Threading.Tasks
+
+open Microsoft.CodeAnalysis.Text
+open Microsoft.CodeAnalysis.CodeFixes
+open Microsoft.CodeAnalysis.CodeActions
+
+[<ExportCodeFixProvider(FSharpCommonConstants.FSharpLanguageName, Name = "AddNewKeyword"); Shared>]
+type internal FSharpAddNewKeywordCodeFixProvider() =
+    inherit CodeFixProvider()
+
+    override __.FixableDiagnosticIds = ["FS0760"].ToImmutableArray()
+
+    override this.RegisterCodeFixesAsync context : Task =
+        async {
+            let title = "Add \"new\" keyword"
```

Comment:
Yes. Could you explain how to do it? (maybe it's worth to add somewhere into DEVGUIDE.md?)